### ISSUE TITLE
YouTube-VIS 2019 train_sub_split.json

### ISSUE THREAD

**Issue by qihao-huang** [JonathonLuiten/TrackEval#58]
In the README [Quickly evaluate on supported benchmarks ](https://github.com/JonathonLuiten/TrackEval#quickly-evaluate-on-supported-benchmarks), the [gt data](https://omnomnom.vision.rwth-aachen.de/data/TrackEval/data.zip) (~150mb) is provided such as YouTube-2019 

> data/gt/youtube_vis/youtube_vis_train_sub_split/train_sub_split.json.

When I look into this gt file, I find the length of videos is 302, same as the YouTube-VIS 2019 validation set provided in [CodaLab](https://competitions.codalab.org/competitions/20128#participate-get-data).  While, this site only provides tranining set annotations (~2238 videos), the validation part is not provided. 

Or, thie data you provied is just part of training set (sample 302 videos from training 2238 videos) ? @JonathonLuiten

Also, thanks for your contribution for Tracking research.
